iarc_forebrain/scripts/Drone.py

```diff
@@ -0,0 +1,518 @@
+import math
+
+import rospy
+import tf
+import tf.transformations
+from ardrone_autonomy.msg import Navdata
+from geometry_msgs.msg import Twist, PoseStamped, Pose, Point, Quaternion
+from iarc_arbiter.msg import RegisterBehavior, VelAlt
+from iarc_main.msg import Roomba
+from numpy import pi
+from std_msgs.msg import String, Empty, Header
+
+
+class Drone:
```

Comment:
Consider renaming to forebrain, central unit, or thereabouts, since all things are essentially about the drone (there's nothing else that we can control).
As of now, layers that control behaviors and sensing are all jumbled up into one class, which makes it difficult to understand what the class actually represents.

Reply:
After talking to Eric, feels ok to leave it as-is.